Getting Started/WalkthroughFirstWPFApp/csharp/ExpenseItIntro.csproj

```diff
@@ -98,12 +30,8 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```

Comment:
I'm not sure you need these.  Have you tried building without them?

Reply:
I used the try-convert tool to convert these projects. I had to make some manual changes in csproj file even after that. Wasn't sure about removing these, so will remove and try building after that